src/tests/mostTraveledCentres.test.js

```diff
@@ -0,0 +1,34 @@
+import Chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+Chai.use(chaiHttp);
+Chai.should();
+
+describe('Get most traveled destination tests', () => {
+  it('It should login successfuly', (done) => {
+    Chai
+      .request(app)
+      .post('/api/v1/auth/signin')
+      .send({
+        email: 'blaiseen@gmail.com',
+        password: 'password',
+      })
+      .end((err, res) => {
+        res.body.should.have.status(200);
+        done();
+      });
+  });
+
+  it('Should return successfully if most traveled centres are found', (done) => {
+    Chai
+      .request(app)
+      .get('/api/v1/trips/most-traveled-centres')
+      .end((err, res) => {
+        res.body.should.have.status(200);
+        res.body.should.have.property('message', 'Trending centres are retrieved successfully');
+        res.body.should.have.property('data');
+        done();
+      });
+  });
+});
```

Comment:
can you try to test like all situations
and if possible please make sure that you test lines you never tested

Reply:
@blaise82 I am diving into that

Reply:
@blaise82 I added the unit tests